WebService/ComparisonEngineUI/ComparisonEngineUI/Data/ListContainer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using ComparisonEngineUI.Models;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ComparisonEngineUI.Data
+{
+    public class ListContainer
+    {
+        private static readonly Lazy<ListContainer> lazy = new Lazy<ListContainer>(() => new ListContainer());
+
+        public static ListContainer Instance { get { return lazy.Value; } }
+
+        public List<Bar> barList;
+        public List<Drink> drinkList;
+        private ListContainer()
```

Comment:
barList and drinkList should be capitalized, no? also whitespace between drinkList and ListContainer constructor